lib/adsb/messages/position.rb

```diff
@@ -14,6 +14,15 @@ def latitude
       def longitude
         @body[71..87].to_i(2).to_f / 131072
       end
+      def odd?
+        @body[53].to_i == 1
+      end
+      def even?
+        @body[53].to_i == 0
+      end
+      def cprflag
```

Comment:
Compact position reports have multiple other flags as well, so I'm not sure if `cprflag` conveys the intent properly. How would you feel about simply `even_or_odd`?